### PR TITLE
Remove old description of the notebook and link to jupyter

### PR DESCRIPTION
**Pull request by jhamrick**
Currently, the first search result on google for "ipython notebook" is http://ipython.org/notebook.html, which describes the notebook as the IPython notebook and makes no reference to Jupyter. In this PR, I have removed the old description of the notebook and provided a link to the Jupyter website.

cc @fperez @willingc @takluyver

### CODE REVIEW

File: notebook.rst [hunk 1]
```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The IPython/Jupyter Notebook
```
Comment (takluyver) [inline notebook.rst]: I think I'd drop 'IPython' from the header entirely, to be clear that it's now the Jupyter notebook.

Though I guess that might make it rank lower for searches on 'IPython notebook'

Comment (jhamrick) [inline notebook.rst]: Yeah, that was my thought -- I wanted to still make this be the most likely hit for "ipython notebook". But I could probably be convinced to remove the "IPython".

Comment (takluyver) [inline notebook.rst]: I could live with it either way. Let's see what @fperez thinks.

**Comment by willingc** [comment]
Perhaps a compromise might be:

 **The Jupyter Notebook (formerly known as IPython notebook)**

**Comment by fperez** [comment]
+1 to Carol's suggestion

**Comment by jhamrick** [comment]
Ok, changed the title.

File: notebook.rst [hunk 2]
```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The Jupyter Notebook (formerly known as IPython notebook)
 ======================
```
Comment (takluyver) [inline notebook.rst]: Have you checked that this still renders as a title? I think rst requires that the over/underline is as long as the text.

Comment (jhamrick) [inline notebook.rst]: Seems to render for me when I build it, though it is quite long:

image

**Comment by willingc** [comment]
Perhaps make it two lines. The Jupyter Notebook has larger font.

**Comment by jhamrick** [comment]
How's this?

image

**Comment by willingc** [comment]
+1

**Comment by jhamrick** [comment]
:tada: http://ipython.org/notebook.html :cake:

**Comment by takluyver** [comment]
Thanks for getting this sorted!

**Comment by fperez** [comment]
Thanks so much!

On Friday, April 1, 2016, Thomas Kluyver <notifications@github.com> wrote:

> Thanks for getting this sorted!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipython-website/pull/113#issuecomment-204642611>
>


-- 
Fernando Perez (@fperez_org; http://fperez.org)
fperez.net-at-gmail: mailing lists only (I ignore this when swamped!)
fernando.perez-at-berkeley: contact me here for any direct mail

**Pull request merged**